### PR TITLE
Enable parallel execution of event-bus tests

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Enable parallel execution of event-bus tests

**Related issue(s)**

See #4297

### CODE REVIEW

File: resources/event-bus/templates/tests/test.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ spec:
             - --publish-event-uri=http://event-bus-publish:8080/v1/events
```
Comment (anishj0shi) [inline resources/event-bus/templates/tests/test.yaml]: it needs to be rebased with master to have the latest changes pulled in

File: resources/event-bus/templates/tests/test.yaml [hunk 2]
```diff
@@ -27,6 +27,7 @@ spec:
             - --publish-event-uri=http://event-publish-service:8080/v1/events
             - --publish-status-uri=http://event-publish-service:8080/v1/status/ready
             - --subscriber-image={{ .Values.global.containerRegistry.path }}/{{ .Values.global.event_bus_tests.dir }}event-bus-e2e-subscriber:{{ .Values.global.event_bus_tests.version }}
```
Comment (anishj0shi) [inline resources/event-bus/templates/tests/test.yaml]: I know this has been there for quiet some time, but shouldn't we reference the image name "event-bus-e2e-subscriber" in values.yaml ? something like this https://github.com/kyma-project/kyma/blob/0e88f482af87dd1e89ad449fec2b6fdfe3eca748/resources/event-bus/charts/event-publish-service/templates/deployment.yaml#L33

Comment (sayanh) [reply resources/event-bus/templates/tests/test.yaml]: IMO, I don't see the benefit. In order to retain consistency with the other charts, I would change https://github.com/kyma-project/kyma/blob/0e88f482af87dd1e89ad449fec2b6fdfe3eca748/resources/event-bus/charts/event-publish-service/templates/deployment.yaml#L33

**Pull request merged**